src/content/posts/2025-pyladiescon-is-back.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "PyLadiesCon is back!"
+meta_title: "PyLadiesCon is back!"
+description: "PyLadiesCon team had a great time at PyCon US 2025 in Pittsburgh, PA"
+date: 2025-06-23T23:00:00Z
+image: "/images/posts/pyladiescon-is-back.png"
+categories: ["Blog Post",]
+authors: ["Georgi"]
+tags: ["python", "community", "conference", "pyladies"]
+draft: false
+---
+
+
+## Save the Date 📅
+Yes! You read it right—**PyLadiesCon** returns December 5–7, 2025. 100% online conference designed for our global community. 
+Talks, workshops, panels, and community fun, all within the comfort of wherever your Wi‑Fi works. 
+
+---
+
+## Website is Live 🌐
+
+The official PyLadiesCon 2025 website is now live! 🎉
+
+Visit [our website](https://conference.pyladies.com/) to explore the new 2025 website, information behind the scene in our documentation or stay tuned for more updates, and announcements on our blog.
+
+---
+
+## What to Expect 🌟
+
+PyLadiesCon is more than just live streams—it's a round-the-clock, multilingual Python conference 🐍🌐 to highlight our underrepresented groups. We know our attendees span the globe, so our program rotates to fit Tokyo mornings ☀️, Berlin afternoons ☕, and late-night coding 🌙 in your secret lair. Whether you fancy catching a keynote in English or a workshop in Spanish, we've got you covered.
```

Comment:
I felt the emoji in the middle of sentences to be a bit distracting 😵 Maybe keep the emoji at the end?

Reply:
updated